lib/onebox/engine/google_docs_onebox.rb

```diff
@@ -4,6 +4,7 @@ module Onebox
   module Engine
     class GoogleDocsOnebox
       include Engine
+      include StandardEmbed
```

Comment:
https://github.com/discourse/onebox/pull/427/files#diff-f791a38b9399c1968a7f840f68457b47e30bc18b7d8bfe4d5d1c550e69bf41c8
Thank you.